test/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ foreach(test ${tests})
   add_executable(${test} EXCLUDE_FROM_ALL ${test}.cc )
   set_target_properties(${test} PROPERTIES
     COMPILE_DEFINITIONS VALHALLA_SOURCE_DIR="${VALHALLA_SOURCE_DIR}/")
-  target_link_libraries(${test} valhalla gtest gmock)
+  target_link_libraries(${test} valhalla gtest gmock -lpthread)
```

Comment:
Why was this suddenly necessary?

Reply:
i made a comment about this in the diff on the other spot where i had to add this: https://github.com/valhalla/valhalla/pull/2278#discussion_r397966748